modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc

```diff
@@ -28,14 +28,14 @@ $ oc create namespace httpbin
 $ oc label namespace httpbin istio-injection=enabled
```

Comment:
Out of scope and totally up to you if you're game for content improvements, but these first two steps could use the addition of "by running the following command:"

Reply:
done